src/components/MenuPanel/MenuPanel.js

```diff
@@ -186,19 +179,19 @@ class MenuPanel extends React.PureComponent {
 }
 
 const AnimatedMenuPanel = ({
-  menuPanelOpened,
+  opened,
+  autoClosing,
   onCloseMenuPanel,
-  autohide,
   ...props
 }) => {
   return (
     <React.Fragment>
       <Spring
         from={{ progress: 0 }}
-        to={{ progress: !!menuPanelOpened }}
+        to={{ progress: Number(opened) }}
```

Comment:
`react-spring` takes care of this no?

Reply:
I don’t think it does? It might work because of JS type coercion, but we better explicitly pass the same type in the `from` and the `to` object for a value.